docs/upgrading.md

```diff
@@ -5,6 +5,17 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.16.1` to `v0.17.0`
+
+- The structure of the build folder has been updated. 
+  The folder is now named `abci_build_{uuid}`, 
+  where `{uuid}` is a 4-character random hash, 
+  replacing the previous format of `abci_build`.
+- The naming convention for service containers has been revised. 
+  Previously, containers were named `{service_name}_{abci|tm}_{index}`. 
+  Now, the format has changed to `{service_name}_{abci|tm}_{uuid}`, 
```

Comment:
hey I just confirmed that these has actually become like `helloworldNfG2_tm_0`. So it should be like this right?
```suggestion
  Now, the format has changed to `{service_name}{uuid}_{abci|tm}_{index}`, 
```
I'm not sure because here the `service_name` is actually cleaned by removing _ and not exactly same as service name.